reconstruction/recoil/src/main/java/org/jlab/service/recoil/recoilCluster.java

```diff
@@ -0,0 +1,212 @@
+package org.jlab.service.recoil;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.jlab.detector.base.DetectorDescriptor;
+import org.jlab.detector.base.DetectorType;
+import org.jlab.geom.prim.Line3D;
+import org.jlab.geom.prim.Point3D;
+
+/**
+ * recoil in-layer cluster
+ * 
+ * @author bondi, devita, niccolai
+ */
+public class recoilCluster extends ArrayList<recoilStrip> {
```

Comment:
According tto Java standards, the name of classes should start with a capital letter..can you make this (and the others) RecoilCluster?